src/asset-swapper/utils/market_operation_utils/types.ts

```diff
@@ -425,7 +425,7 @@ export type OptimizedMarketOrder =
     | OptimizedMarketOrderBase<NativeRfqOrderFillData>;
 
 export interface GetMarketOrdersRfqOpts extends RfqRequestOpts {
-    rfqClient?: IRfqClient;
+    rfqClient?: RfqClient;
```

Comment:
Any reasons to prefer the implementation over interface? If we really don't want to use `IRfqClient` maybe consider to remove it from `irfq_client.ts`.

Reply:
removing